README.md

```diff
@@ -360,9 +360,9 @@ such as `csv`, `avro` etc.
 HarbourBridge accepts the following options for --target-profile,
 specified as "key1=value1,key2=value,..." pairs:
 
-`dbname` Specifies the name of the Spanner database to create. This must be a
-new database. If dbname is not specified, HarbourBridge creates a new unique
-dbname.
+`dbName` Specifies the name of the Spanner database to create. This must be a
```

Comment:
Nit: We should probably take these cleanups as independent PR's going forward

Reply:
Agreed, let me see if I can split them anyways.